src/TwigComponent/src/Command/TwigComponentDebugCommand.php

```diff
@@ -85,6 +87,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $components = $this->findComponents();
         $this->displayComponentsTable($io, $components);
+        $this->displayDetailsAboutConfiguration($io);
```

Comment:
shouldn't the configuration appear first, above components?

Reply:
btw, worth being a separate PR IMO (without the new setting of course), may get merged faster than this one

Reply:
Yes it will be better if you can split this @Halleck45 thx!